apps/nextra/pages/en/build/create-aptos-dapp/templates/token-minting-dapp.mdx

```diff
@@ -15,118 +17,136 @@ The Token minting dapp template provides 3 pages:
-```bash 
-npm run dev
+If you haven't set up an account, you can create one with an Aptos wallet like [Petra](https://petra.app/docs/use#create-a-new-account). Make sure to set up the account on the same network as your template project.
+
+For example:
+```bash filename=".env"
+VITE_APP_NETWORK=testnet
+VITE_FA_CREATOR_ADDRESS=0x71e3cdf1ef86d575ae38c7c2d19f22b8c88eb2b15082691782e5b8c8da401568
 ```
 
-### Publish the Move module
+### On the project root folder, run the below command to publish the contract.
 
-To create a new asset, we will need to publish the Move module on-chain.
+```bash
+npm run move:publish
+```
+<Callout type="info">
+You will have to confirm the gas fee in the console.
+</Callout>
 
-The smart contract is built in a way that only the admin or a specific defined account can create a new asset.
+This command will publish the contract to chain.
 
-Create or find the account you want to create an asset with. If you haven't created an account - simply use a Wallet, for example [Petra](https://petra.app/), to quickly [create an account](https://petra.app/docs/use#create-a-new-account).
+</Steps>
 
-1. Run `npm run move:init` - a command to initialize an account to publish the Move contract. When you run that command it will generate a new `.aptos/config.yaml` file that holds a profile with the account private key, account address, and network configuration.
-```bash 
-npm run move:init
-```
 
-2. To set who can create a new asset, edit the `.env` file and update `VITE_FA_CREATOR_ADDRESS` to be the address of the account allowed to create assets.
-```bash 
-// .env file
-VITE_FA_CREATOR_ADDRESS=<create-asset-account-address>
-```
+## Publishing the asset on-chain
+<Steps>
 
-3. Run `npm run move:publish` to publish the contract. When you run that command it will save the module address to `VITE_MODULE_ADDRESS` in the `.env` file.
-```bash 
-npm run move:publish
+### Run the app and open the preview.
+
+```bash
+npm run dev
 ```
 
-### Upgrade the Move module
+### In the top-right corner, click "Connect a Wallet". 
+Use the same account that you set in `.env` for the `VITE_FA_CREATOR_ADDRESS`. 
+Or, if you use a different account, set the `VITE_FA_CREATOR_ADDRESS` to the account you sign in with and republish the contract.
 
-When you make changes to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you can run `npm run move:upgrade` to upgrade previous published module. When you make incompatible changes, you can run `npm run move:publish` to publish to a new address.
+<Callout type="warning">
+Do not use the "Google Account" option if it is available. The account generated from that will not work as the `VITE_COLLECTION_CREATOR_ADDRESS`. Use a wallet created with an Aptos wallet provider such as [Petra](https://petra.app/docs/use#create-a-new-account).
```

Comment:
I'm a little confused with what this means

Reply:
simply says you can't create a collection with a Aptos Connect account